src/functionalTest/java/uk/gov/hmcts/reform/jps/StepDefinitions.java

```diff
@@ -201,7 +209,7 @@ public void theResponseReturnsTheMatchingSittingRecords() {
             .body("sittingRecords[0]", Matchers.hasKey("accountCode"))
             .body("sittingRecords[0]", Matchers.hasKey("fee"))
             .body("sittingRecords[0].am",equalTo("AM"))
```

Comment:
Should AM assertion be as below?

Reply:
Yes, this is because of a bug that we have for the search endpoint, it is returning AM instead of true. This is going to be fixed as part of [IJPS-56](https://tools.hmcts.net/jira/browse/IJPS-56)